src/components/ReactIcon/ReactIcon.tsx

```diff
@@ -0,0 +1,13 @@
+import { IconContext } from 'react-icons';
+import { PropsWithChildren } from 'react';
+
+interface ReactIconProps extends PropsWithChildren {
+  value: object;
+}
+export const ReactIcon = ({ value, children }: ReactIconProps) => {
```

Comment:
keep 1 empty line between each interface, function, component - whatever. For better readability